Library/Formula/artifactory-cli-go.rb

```diff
@@ -0,0 +1,13 @@
+class ArtifactoryCliGo < Formula
+    desc "Artifactory CLI provides a command line interface for uploading and downloading artifacts to and from Artifactory."
+    homepage "https://github.com/JFrogDev/artifactory-cli-go"
+    url "https://bintray.com/artifact/download/jfrog/artifactory-cli-go/1.2.0/artifactory-cli-mac-386/art"
```

Comment:
Can this be built from source instead? We don't accept prebuilt binaries in this repository.